src/main/java/com/sda/project/management/model/User.java

```diff
@@ -101,10 +111,13 @@ public void setProjectAccessList(Set<ProjectAccess> projectAccessList) {
     public String toString() {
         return "User{" +
                 "id=" + id +
-                ", username='" + username + '\'' +
                 ", email='" + email + '\'' +
+                ", password='" + password + '\'' +
+                ", firstName='" + firstName + '\'' +
+                ", lastName='" + lastName + '\'' +
                 ", displayName='" + displayName + '\'' +
                 ", roles='" + roles + '\'' +
+                ", projectAccessList=" + projectAccessList +
```

Comment:
do not add lists of objects to the toString() method to prevent access exceptions